sklearn/ensemble/tests/test_bagging.py

```diff
@@ -65,7 +65,7 @@ def test_classification():
 
     for base_estimator in [None,
                            DummyClassifier(),
-                           Perceptron(),
+                           Perceptron(tol=1e0),
```

Comment:
I changed the tests to use the future parameters and avoid a FutureWarning.
I didn't use the future default values to avoid a ConvergenceWarning